desktop/core/src/desktop/redaction/tests.py

```diff
@@ -332,6 +335,22 @@ def test_real_rules(self):
     for message, redacted_message in messages:
       assert_equal(redacted_message, policy.redact(message))
 
+  def test_unicode_strings(self):
+    path = get_path('real-1.json')
```

Comment:
I see, it is a test file already there

Reply:
yes real-1.json are the redaction rules defined json
https://github.com/cloudera/hue/blob/master/desktop/core/src/desktop/redaction/test_data/real-1.json

reusing the same file as it suffice the basic neccessity :)